plugin/adsk/scripts/USDMenuProc.mel

```diff
@@ -24,302 +24,15 @@ proc string expandToSelection(string $obj)
-    window -e -height 260 $gOptionBox;
-}
-
 // On right click in the viewport, either through selection or through a pick
 // hit, Maya will look for a procedure called $runTimeName + "MenuProc".
 // and call it.  When the selection or pick hit is a USD prim, this calls the
 // following procedure.
 global proc USDMenuProc(string $parent, string $obj)
 {
-    if (!hasPrimUpdater())
-        return;
-        
     $obj = expandToSelection($obj);
-    if (size($obj) != 0)
-    {
-        popupMenu -e -dai $parent;
-        setParent -menu $parent;
-
-        setParent -menu ..;
-        if (canEditAsMaya($obj)) {
-            menuItem -label `getMayaUsdString("kEditAsMayaData")` -image "edit_as_Maya.png" -command ("{waitCursor -state 1; mayaUsdMenu_editAsMaya \"" + $obj + "\"; waitCursor -state 0;}");
-            menuItem -command ("{mayaUsdMenu_EditAsMayaDataOptions \"" + $obj + "\";}") -optionBox true;
-        }
-        if (canDupAsMaya($obj)) {
-            menuItem -label "Duplicate As Maya Data" -command ("{waitCursor -state 1; mayaUsdMenu_duplicate \"" + $obj + "\"; waitCursor -state 0;}");
-        }
-
-        menuItem -divider true;
-
-        // Material-related options should only be shown if the right-clicked object is of a type known to support material bindings.
-        int $allowMaterialFunctions = `mayaUsdMaterialBindings $obj -canAssignMaterialToNodeType true`;
-        if ($allowMaterialFunctions)
-        {
-            // "Assign New Material" from a list of materials provided by the renderers.
-            menuItem -subMenu true -tearOff false -label "Assign New Material";
-            {
-                mayaUsdMenu_addNewMaterials($obj);
-            }
-            setParent -menu ..;
-
-            // Only show if we have materials in the stage.
-            string $materials[] = `mayaUsdGetMaterialsInStage $obj`;
-            if (size($materials) > 0)
-            {
-                // "Assign Existing Material" from a list of materials in the selected item's stage.
-                menuItem -subMenu true -tearOff false -label "Assign Existing Material";
-                {
-                    mayaUsdMenu_addExistingMaterials($obj);
-                }
-                setParent -menu ..;
-            }
-
-            // Only show the following entry if the object actually has a material assigned.
-            int $hasMaterialBindingAPI = `mayaUsdMaterialBindings $obj -hasMaterialBinding true`;
-            if ($hasMaterialBindingAPI)
-            {
-                // Unassign (aka "unbind") the material from the selected object.
-                menuItem -label "Unassign Material" -command ("{waitCursor -state 1; mayaUsdMenu_unassignMaterial \"" + $obj + "\"; waitCursor -state 0;}");
-            }
-        }
-    }
+    if (size($obj) == 0)
+        return;
 
-    // Allow the user to define a proc with additional menu items
-    if (`exists USDUserMenuProc`)
-    {
-        USDUserMenuProc($parent,$obj);
-    }
```

Comment:
This needs to stay. It was added specifically at a customer request and thus might be used by them.